src/components/ScriptEditor.jsx

```diff
@@ -176,6 +176,7 @@ class ScriptEditor extends React.Component {
       <div>
         <div style={styles.editor} onClick={this.focus}>
           <Editor
+            webDriverTestID='editorid' // Will render as data-testid
```

Comment:
Is webDriverTestID used anywhere else? If not, potentially confusing.

Reply:
This is something I found in the draft-js project:
https://github.com/facebook/draft-js/issues/325

The challenge here is much like the DatePicker problem in that we cannot directly assign an attribute to the input element because it's not our code. So we're forced to use what we're given. I'll try to poke around a little more but I don't think we can avoid the confusion. Perhaps a comment with a link to the above issue to help those who come after us?

Reply:
No it is not used anywhere else. This was something I found in the draft-js project to workaround the fact that we cannot directly assign an attribute to the editor.

https://github.com/facebook/draft-js/issues/325

It works just as well to use their class as a selector and potentially avoid confusion.